tests/components/asuswrt/test_sensor.py

```diff
@@ -44,6 +44,11 @@
 MOCK_CURRENT_TRANSFER_RATES = [20000000, 10000000]
 
 
+def get_devices():
+    """Return mock devices for test."""
+    return MOCK_DEVICES.copy()
```

Comment:
Alternatively, we could just create the dict here and remove the constant.

Reply:
But test require that values in the dict is changed

Reply:
`MOCK_DEVICES` is a constant so we should never modify it. We may need to refactor a bit more to solve this.

Reply:
Refactored adding MockDevice class.